### DISCUSSION TITLE
validator.get_root_schema

### DISCUSSION THREAD

**Discussion by sergeniously** [pboettch/json-schema-validator#289] [Ideas]
Why is there no get_root_schema function to get a complete resolved root schema?
It would be very useful for (as example) building user interfaces in GUI applications.

**Comment by pboettch** [comment]
You mean a function to create a json-schema after everything has been loaded and all references have been resolved? If so, nobody has ever requested this.

**Reply by sergeniously** [reply]
Yes, I mean exactly what you said. Is it easy to implement?

**Reply by pboettch** [reply]
A serialization of the different validator-classes to json-schema. It's feasible, but _I_ would have to time to do that in the near future.
